almond/build.yaml

```diff
@@ -2,8 +2,6 @@ build_from:
   aarch64: ghcr.io/home-assistant/aarch64-base-debian:buster
   amd64: ghcr.io/home-assistant/amd64-base-debian:buster
   armv7: ghcr.io/home-assistant/armv7-base-debian:buster
-codenotary:
-  signer: notary@home-assistant.io
-  base_image: notary@home-assistant.io
```

Comment:
Actually wait, I thought this was a rebase issue but it looks like this section was specifically removed from the `build.yaml` file. Is there a reason this information was removed?